### ISSUE TITLE
Dynamically changing data-max screws with knob size 

### ISSUE THREAD

**Issue by ojsglobal** [aterrien/jQuery-Knob#270]
When I run this:

$('#moves').trigger('configure', 
{	
'max': 500
});

$('#moves').trigger('change');

it completely loses the original size, and shrinks it down to tiny.

Any ideas?

**Comment by zxfrank** [comment]
answered in #187

**Comment by ojsglobal** [comment]
No, this doesn't resolve the issue at all, in fact it demonstrates it.
